src/components/LoginLoginLogin.vue

```diff
@@ -0,0 +1,115 @@
+                  >
+                    <div class="form-outline form-white mb-4">
+                      <label class="form-label"
+                        >Username</label
+                      >
+                      <div>
+                        <input
+                          type="text"
+                          v-model="username"
+                          class="form-control"
+                          required
+                        />
+                        <div v-if="usernameError" style="color: red;">
+                          {{ usernameError }}
+                        </div>
+                      </div>
+                    </div>
+                    <div class="form-outline form-white mb-4">
+                      <label class="form-label"
+                        >Password</label
+                      >
+                      <div>
+                        <input
+                          type="password"
+                          v-model="password"
+                          class="form-control"
+                          required
+                        />
+                        <div v-if="passwordError" style="color: red;">
+                          {{ passwordError }}
+                        </div>
+                      </div>
+                    </div>
+                    <button
+                      class="btn btn-outline-light btn-lg px-5"
+                      type="submit"
+                    >
+                      Login
+                    </button>
+                  </form>
+                </div>
+                <div>
+                  <p class="mb-0">
+                    Don't have an account?
+                    <a href="#!" class="text-white-50 fw-bold">Sign Up</a>
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+  </div>
+</template>
+<script>
+import axios from "axios";
+import { mapGetters } from "vuex";
+export default {
+  data() {
+    return {
+      token: null,
+      username: "",
+      password: "",
+      usernameError: '',
+      passwordError: ''
+    };
+  },
+  methods: {
+    async submitForm() {
+      try {    
+        const formData = {
+          username: this.username,
+          password: this.password,
+        };
+        const TOKEN = await axios.post("/api/token/", formData);
+        localStorage.setItem("user-token", TOKEN.data.access);
+        this.token = localStorage.getItem("user-token");
+        const USER = await axios.get("/users/", {
+          headers: {
+            Authorization: `Bearer ${this.token}`,
+          },
+        });
+        localStorage.setItem("username", USER.data.results[0].username);
+        this.$store.dispatch("userToken", this.token);
+        this.$router.push("/").catch(() => {});
+      } catch {
+        this.usernameError = this.username == '' ? 'Please enter a username !' : 'Invalid username !';
```

Comment:
Lines of code that check for the existence of username and password are supposed to be at the beginning of the submitForm method and if they do not exist we should not allow further execution of the submitForm method. But I am wondering, why did you install `vuelidate` libraries if you are not using them. Existance of these fields is supposed to be checked with `vuelidate` library and not manually, that is the purpose of the library. Please make use of the library.
Also, you are handling only an Invalid username or password error, what about server error? Please handle them as well, and you will be able to do so only if you check the error in the catch block.